rustfmt.toml

```diff
@@ -10,8 +10,6 @@ newline_style = "Unix"
 # Misc
 chain_width = 80
 spaces_around_ranges = false
-binop_separator = "Back"
-reorder_impl_items = false
 match_arm_leading_pipes = "Preserve"
```

Comment:
I'd prefer not to change this and leave this the same as substrate, as we sometimes do upstream diffs.

Reply:
Although, I agree that this is a weird choice of them.

Reply:
Alright, I'll revert this again. My intention was to have it the same way as in the worker. But you're saying these differences are intentional?